packages/api-headless-cms/src/utils/createModelField.ts

```diff
@@ -50,3 +53,46 @@ export const createModelField = (params: CreateModelFieldParams): CmsModelField
         renderer
     };
 };
+
+export const createModelFieldPlugin = (params: CreateModelFieldParams): CmsModelField => {
```

Comment:
Actually, I would leave out the `Plugin` suffix, because the fn result is not a plugin. It's just model field object.

Which is different from other utilities, which actually _do_ return a plugin object / instance.

Reply:
sounds good, thank you @adrians5j!